image/base/install-rust

```diff
@@ -5,7 +5,7 @@ set -e
 export RUSTUP_HOME=/usr/local/rustup
 export CARGO_HOME=/usr/local/cargo
 export PATH=/usr/local/cargo/bin:$PATH
-export RUST_VERSION=1.68.0
+export RUST_VERSION=1.75.0
```

Comment:
Had to update to the latest version of rust to be able to compile `selenium-manager`.